python-frontend/src/main/java/org/sonar/plugins/python/api/symbols/ClassSymbol.java

```diff
@@ -39,4 +39,7 @@
 
   @Beta
   boolean isOrExtends(ClassSymbol other);
+
+  @Beta
+  boolean canBeOrExtend(String fullyQualifiedClassName);
```

Comment:
What about using this new method in other places? Are you going to open a new PR for that?

Reply:
Yes, good point. I think we have a few rules that can benefit from this API